client/src/components/transformation/merge/SourceMergeOptions.jsx

```diff
@@ -115,7 +132,7 @@ class SourceMergeOptions extends Component {
       mergeColumn: null,
       aggregationColumn: null,
       aggregationDirection: 'DESC',
-      mergeColumns: [],
+      mergeColumns: Immutable.fromJS([]),
```

Comment:
I guess we could do Immutable.List() which in my mind would be a bit more attractive.